components/channel_header/channel_header.js

```diff
@@ -1,6 +1,5 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
-/* eslint-disable react/no-string-refs */
```

Comment:
thanks for taking care of this, seems to be a leftover we overlooked when reviewing the string refs campaing 👍